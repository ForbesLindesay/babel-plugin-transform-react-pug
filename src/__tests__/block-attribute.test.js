import testHelper from './test-helper';

testHelper(__dirname + '/block-attribute.input.js');
