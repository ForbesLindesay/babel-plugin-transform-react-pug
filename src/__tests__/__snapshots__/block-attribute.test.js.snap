// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JavaScript output: transformed source code 1`] = `
"module.exports = (
  <menu icon=<img />>
    <button />
  </menu>
);
"
`;

exports[`html output: generated html 1`] = `
<menu
  icon={<img />}
>
  <button />
</menu>
`;

exports[`static html output: static html 1`] = `"<menu icon=\\"[object Object]\\"><button></button></menu>"`;
